x/adminmodule/abci.go

```diff
@@ -95,6 +95,7 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 
 		keeper.SetProposal(ctx, proposal)
 		keeper.RemoveFromActiveProposalQueue(ctx, proposal.Id)
+		keeper.AddToArchive(ctx, proposal)
```

Comment:
why do we still need the active proposals handling in the EndBlock? I thought, since we switch to instant proposals execution, we can remove this logic from the adminmodule's EndBlock